pkgs/by-name/xm/xmr-btc-swap/package.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, protobuf
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "xmr-btc-swap";
+  version = "0.13.1";
+
+  src = fetchFromGitHub {
+    owner = "comit-network";
+    repo = pname;
+    rev = version;
+    hash = "sha256-vR4Xu+GmEC18QHhu+20pQIjKUD/WFk4q0CuHXRINGxY=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "bitcoin-harness-0.2.1" = "sha256-enRFvcZKHZuchIYPrsKIPujMEwrzwErMN4P0vobVRl4=";
+      "jsonrpc_client-0.7.1" = "sha256-avWNDE6O7Afhqd2OiWD23I7gSHhVRVg3WREyGpgrUaU=";
+      "monero-0.12.0" = "sha256-HBtA7FhrvFOOTv48UK5yMuQV4etc8ULQA0L9KZNfakI=";
+    };
+  };
+
+  postPatch = ''
+    rm swap/build.rs
+  '';
+
+  nativeBuildInputs = [ protobuf ];
+
+  env.VERGEN_GIT_DESCRIBE = version;
+  doCheck = false;
```

Comment:
This needs a very short comment on why the tests are disabled

Reply:
I don't really know why the test failed. It requires docker and fails after a long time with "failed to create docker network".